packages/flutter_tools/test/create_test.dart

```diff
@@ -133,16 +135,28 @@ void main() {
       Cache.flutterRoot = '../..';
       final CreateCommand command = new CreateCommand();
       final CommandRunner<Null> runner = createTestCommandRunner(command);
-      final File existingFile = fs.file("${temp.path.toString()}/bad");
+      final File existingFile = fs.file("${projectDir.path.toString()}/bad");
       if (!existingFile.existsSync())
-        existingFile.createSync();
+        existingFile.createSync(recursive: true);
       try {
         await runner.run(<String>['create', existingFile.path]);
         fail('expected ToolExit exception');
       } on ToolExit catch (e) {
         expect(e.message, contains('file exists'));
       }
     });
+
+    testUsingContext('fails when invalid package name', () async {
+      Cache.flutterRoot = '../..';
+      final CreateCommand command = new CreateCommand();
+      final CommandRunner<Null> runner = createTestCommandRunner(command);
+      try {
+        await runner.run(<String>['create', 'invalidName']);
+        fail('expected ToolExit exception');
```

Comment:
Maybe change [this](https://github.com/flutter/flutter/blob/83d411f97962d08671ecca5a9df7c59203158265/packages/flutter_tools/test/src/common.dart#L66) to accept a message parameter, then use:

```
expect(runner.run(<String>['create', 'invalidName']), throwsToolExit(...))
```